templates/vue-common/mixins/ListMixin.js

```diff
@@ -33,16 +33,25 @@ export default {
 
   methods: {
     onUpdateOptions(props) {
-      const { page, itemsPerPage, sortBy, descending, totalItems } = props;
+      const { page, itemsPerPage, sortBy, sortDesc, descending, totalItems } = props;
       let params = {
         ...this.filters
       };
       if (itemsPerPage > 0) {
         params = { ...params, itemsPerPage, page };
       }
+      
+      let sortDescVuetify = false;
```

Comment:
if think you can skip this block and only change the line appending the params

Reply:
remove this block of code cause it handled l. [51](https://github.com/api-platform/client-generator/pull/215/files#diff-b13a09535d55bd6082abfb8cf94afecfR51)